admin/securedrop_admin/__init__.py

```diff
@@ -642,7 +642,7 @@ def check_for_updates(args):
 
 
 def get_release_key_from_keyserver(args, keyserver=None, timeout=45):
-    gpg_recv = ['timeout', str(timeout), 'gpg', '--recv-key']
+    gpg_recv = ['timeout', str(timeout), 'gpg', '--batch', '--recv-key']
```

Comment:
I think we should add `--no-tty` also to be on the safe side (so that gpg doesn't try to write output to the tty, which doesn't exist). From the gpg docs:

```
--no-tty
     Make sure that the TTY (terminal) is never used for any output. This option is needed in some cases because 
     GnuPG sometimes prints warnings to the TTY if if --batch is used.
```

cc @kushaldas @zenmonkeykstop

Reply:
Thanks once again to catch the specialties and tame GnuPG. Updated per your suggestion.

Reply:
@KwadroNaut I think the test failure was due to keyserver flakiness. I've had the test suite pass several times. The only consistent problem is a linting problem; flake8 wants the second line of arguments to start after the opening bracket of the list: `securedrop_admin/__init__.py:646:13: E128 continuation line under-indented for visual indent`. If you have time to slide that over and push again, great, otherwise a maintainer will just do it later in the week. Thanks again!